Readded package: Aurora.Aurora version v0.8.1

OfficialEsco opened the pull request:
Gets stuck installing VC++, waiting for microsoft/winget-cli#163

- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

![image](https://user-images.githubusercontent.com/15158490/127676457-cf39305c-63f2-4924-9caf-83a95418af64.png)

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22900)

OfficialEsco committed:
Metadata: Aurora.Aurora version v0.8.1

OfficialEsco commented:
.

arjun-msft commented:
/azp run

OfficialEsco commented:
> Gets stuck installing VC++, waiting for microsoft/winget-cli#163

This needs Blocking-Issue.

stephengillie commented:
/AzurePipelines run

OfficialEsco committed:
Update Aurora.Aurora.installer.yaml

stephengillie commented:
/AzurePipelines run

stephengillie commented:
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 
> 2023-12-27 12:09:50.593 [CLI ] ShellExecute installer failed: 1 
> 2023-12-27 12:09:50.403   Exception message:                           Internal error: Cannot expand "pf64" constant on this version of Windows.

(Automated response - build 490.)

stephengillie commented on manifests/a/Aurora/Aurora/v0.8.1/Aurora.Aurora.installer.yaml:
```diff
@@ -0,0 +1,28 @@
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```
```suggestion
  Architecture: x64
```
From the error, I'm guessing that this is a 64-bit application.

OfficialEsco committed:
Apply suggestions from code review

Weird, the uninstall key points to the x86 registry but its actually installed to `C:\Program Files\Aurora`
```
Name          : Aurora v0.8.1
Publisher     : Anton Pupkov
Version       : v0.8.1
ProductCode   : {9444602B-C5D8-4EF5-9D5B-E76D06B53C71}_is1
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\{9444602B-C5D8-4EF5-9D5B-E76D06B53C71}_is1
Uninstall     : winget uninstall --id "{9444602B-C5D8-4EF5-9D5B-E76D06B53C71}_is1"
```

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

stephengillie commented:
/AzurePipelines run

stephengillie commented:
Automatic Validation ended with:
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/e862bc8c-e91e-4fa9-8077-89f9a3772c26)

OfficialEsco commented:
works on my device 👀
![image](https://github.com/microsoft/winget-pkgs/assets/15158490/c7f32c09-b216-4114-a945-b67b81f95dae)

stephengillie commented:
@wingetbot waivers Add Validation-Unattended-Failed

OfficialEsco commented:
@wingetbot run

stephengillie commented:
Hi @OfficialEsco,

This PR appears to modify 2 versions: `v0.8.1` and `0.8.1` - is this an artifact of how GitHub is displaying the PR? 
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/cb5fda41-7091-4152-b7df-ef74b9ae7f3a)

OfficialEsco commented:
Yes, we're currently on a branch before Aurora got deleted for a reason, the original PR was for fixing the DisplayVersion so its correctly matching.
I could try to rebase if you wish, but in theory only the add's will be merged as the deleted already exist

OfficialEsco committed:
Update Manifest

Trenly commented:
@wingetbot run
